VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/dao/LoginDao.java

```diff
@@ -0,0 +1,38 @@
+package edu.vt.cs.vtcare.vtcareservice.dao;
+import edu.vt.cs.vtcare.vtcareservice.db.VTCareJDBC;
+
+import java.sql.*;
+
+public class LoginDao {
```

Comment:
This is an inappropriate way of creating Dao. Login is not an entity in the system. And dao objects are created per "entity" in the system. It doesn't make any sense to have a LoginDao in the service. 
This method should go in appropriate user class for example if your are authenticating provider then in ProviderDao, if patient then in PatientDao, and likewise for AdminDao. 
Delete this file and refractor this code into ProviderDao.

Reply:
I beg to differ, it reduces code duplicates by making login functionalities in one place. Since login is a single operation, it will not violate the principle of single responsibility and thus no need to make changes. It is a design choice and is better as an approach for this project.

Reply:
Again, it's not a question of how you feel about it. If we go like that then we would end up with spaghetti code. We create Daos for entities. And this class is not valid. Please delete this class and refractor the code as suggested. I am sorry but personal design choices can't be taken in place of established design practices.